src/main/java/com/kneelawk/extramodintegrations/tconstruct/Util.java

```diff
@@ -1,10 +1,25 @@
 package com.kneelawk.extramodintegrations.tconstruct;
 
+import java.util.ArrayList;
+import java.util.List;
 import dev.emi.emi.api.stack.EmiStack;
 import io.github.fabricators_of_create.porting_lib.fluids.FluidStack;
+import net.minecraft.text.Text;
+import net.minecraft.util.Formatting;
+import slimeknights.mantle.fluid.tooltip.FluidTooltipHandler;
 
 public class Util {
     public static EmiStack convertFluid(FluidStack fluidStack) {
         return EmiStack.of(fluidStack.getFluid(), fluidStack.getTag(), fluidStack.getAmount());
     }
+
+    public static List<Text> getFluidTiCTooltip(FluidStack fluidStack) {
+        List<Text> tooltip = new ArrayList<Text>();
+        if (FluidTooltipHandler.appendMaterialNoShift(fluidStack.getFluid(), fluidStack.getAmount(), tooltip)) {
```

Comment:
this check doesn't seem to be working correctly
![image](https://github.com/Kneelawk/extra-mod-integrations/assets/43531981/ed8d8532-cb96-4d73-aca3-0af0b236b8a4)

Reply:
@Abbie5 its a bug of tcon's mantle library. I also checked their source, the check isn't very sophisticated, so it sometimes work, sometimes not. I don't think without parsing text and checking if output matches a regex, there is no way to fix it easily.